oci/cas/cas.go

```diff
@@ -32,13 +32,18 @@ import (
 )
 
 const (
-	// BlobAlgorithm is the name of the only supported digest algorithm for blobs.
-	// FIXME: We can make this a list.
-	BlobAlgorithm = digest.SHA256
+	// DefaultBlobAlgorithm is the default supported digest algorithm for blobs
+	DefaultBlobAlgorithm = digest.SHA256
```

Comment:
Deprecate in favor of go-digest's [`Canonical`][1]?

[1]: https://github.com/opencontainers/go-digest/blob/v1.0.0-rc1/algorithm.go#L39

Reply:
> Deprecate in favor of go-digest's `Canonical`?

Spun off into #197.